Pdf2Video_Converter/requirements.txt

```diff
@@ -0,0 +1,19 @@
+certifi==2020.12.5
```

Comment:
this has a lot of unnecessary modules such as requests, numpy, tqdm. Only include those modules which are used in your script